src/routes/users.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-console */
+import express from 'express';
+import { User } from '../database/models';
+
+const officeRoute = express.Router();
```

Comment:
still called office route. And this route is not currently needed. No need for any dummy API. Especially this one is a security risk as it is sending all user data without the need for any authentication or authorization